packages/victory-area/package.json

```diff
@@ -27,8 +28,138 @@
   "peerDependencies": {
     "react": ">=16.6.0"
   },
+  "devDependencies": {
+    "victory-chart": "^36.5.3"
+  },
   "scripts": {
-    "version": "yarn run nps build-libs && nps build-dists"
+    "build": "wireit",
+    "build:lib": "wireit",
+    "build:lib:esm": "wireit",
+    "build:lib:cjs": "wireit",
+    "build:dist": "wireit",
+    "build:dist:dev": "wireit",
+    "build:dist:min": "wireit",
+    "format": "wireit",
+    "lint": "wireit",
+    "jest": "wireit"
   },
-  "sideEffects": false
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:lib",
+        "build:dist"
+      ]
+    },
+    "build:lib": {
+      "dependencies": [
+        "build:lib:esm",
+        "build:lib:cjs"
+      ]
+    },
+    "build:lib:esm": {
+      "command": "nps build:lib:esm",
+      "files": [
+        "src/**",
+        "!src/**/*.test.*",
+        "../../.babelrc.build.js"
+      ],
+      "output": [
+        "es/**"
+      ],
+      "dependencies": [
+        "../victory-core:build:lib:esm",
```

Comment:
In testing this out, I noticed that any changes to `victory-core` will result in a full-rebuild of all other packages.  This is not so great.

So let's talk about these dependency chains.
Technically, for `victory-area` to build, we don't actually need `victory-core` built.  So if we removed this dependency, that would fix the "full rebuild" problem.

However, obviously `victory-core` is a dependency that needs to be built for this package to work. So, where's the RIGHT spot to put this dependency?

It seems like `victory-core` is a "runtime" dependency.  It's needed for `jest`, and maybe it's needed for `build:dist`?

Reply:
(and TypeScript will probably have its own full dependency chain on other types, but not necessarily on `build:esm` etc)

Reply:
I think this is resolvable now? 

For an import of `victory-area` to be correct in dev (tests, dev server, etc.), if a transitive victory dep has changes, we need the updated transpiled files. Fortunately, once the transitive dep (like `victory-core`) is rebuilt once, it's cached and shouldn't be a big deal elsewhere.